tabs/_common.py

```
@@ -1,52 +1,44 @@
-# No imports needed - this module provides pure data functions
+# แก้ไขไฟล์ tabs/_common.py
+# คงคอมเมนต์และโครงสร้างเดิมไว้ทั้งหมด
+
+def wrap_with_container(content): # ลบ Type hint TagChild ออกเพื่อเลี่ยงการ import shiny ด้านบน
```

Ruff (0.14.10)

Missing return type annotation for private function `wrap_with_container`

(ANN202)